src/Kestrel.Core/Http2Limits.cs

```diff
@@ -14,6 +14,7 @@ public class Http2Limits
         private int _maxStreamsPerConnection = 100;
         private int _headerTableSize = (int)Http2PeerSettings.DefaultHeaderTableSize;
         private int _maxFrameSize = (int)Http2PeerSettings.DefaultMaxFrameSize;
+        private int _initialWindowSize = 1024 * 96; // Larger than the default 64kb
```

Comment:
In which interop tests do we need this limit to be bigger?

Reply:
It's not that we need it to be bigger, it's that we need it to be a value other than the default so that some of the extra logic runs like sending a window update at the start of the connection. I want that active at least for preview3 so we make sure it works with clients.

Reply:
@halter73 because we don't have any interop tests to speak of, I changed it so it would get exercised while doing normal usage during preview3.

Note IIS & Http.Sys use a default of 1mb. That seemed a bit much for starters, but I expect we'll want to make this size larger when we start perf testing.